### PR TITLE
object/replicate: Add signature to the replication RPC

### PR DESCRIPTION
**Pull request by carpawell**
I am not sure about `create` naming.

### CODE REVIEW

File: object/service.proto [hunk 1]
```diff
@@ -721,10 +721,20 @@ message ReplicateRequest {
...
 }
 
 // Replicate RPC response
 message ReplicateResponse {
   // Operation execution status with one of the enumerated codes.
   neo.fs.v2.status.Status status = 1;
+
+  // Signature of replicated object. Must be attached if a request was
+  // made with `create` flag set.
+  bytes object_signature = 2;
```
Comment (cthulhu-rider) [inline object/service.proto:739]: if this is not `neo.fs.v2.refs.Signature`, we should clarify the scheme in docs

**Comment by cthulhu-rider** [comment]
> I am not sure about `create` naming.

to me it's unclear too. Lucky we can freely change it later. Now i cannot suggest nothing better than straightforward `attach_signature`

**Review by cthulhu-rider — APPROVED** [review]

**Review by roman-khimov — APPROVED** [review]

**Comment by carpawell** [comment]
@cthulhu-rider, @roman-khimov, renamed the field name and specified signature algorithm. Do you have any preferences about the scheme? Do we have any rules of when we try to use one or another?

File: object/service.proto [hunk 2]
```diff
@@ -721,10 +721,21 @@ message ReplicateRequest {
 
   // Signature of `object.object_id.value` field.
   neo.fs.v2.refs.Signature signature = 2;
+
+  // Optional flag that requires server side to attach signature of just
+  // replicated object to ensure it has been received correctly. Signature
+  // must be calculated with an exposed to network map public key that
```
Comment (cthulhu-rider) [inline object/service.proto:727]: mb im picky, but `signature calculated` and `public key` don't go well together

**Review by cthulhu-rider — APPROVED** [review]

**Review by cthulhu-rider — APPROVED** [review]

**Comment by carpawell** [comment]
@roman-khimov, have you something to add here?

**Comment by roman-khimov** [comment]
I'm mostly concerned about ersatz-chain test results.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**